### ISSUE TITLE
Tooltip: Escape key not dismissing tooltip (WCAG 2.1 1.4.13)

### ISSUE THREAD

**Issue by kishore5242** [primefaces/primereact#6680]
### Describe the bug

The tooltip displayed may obscure adjacent contents and user may want to dismiss it by pressing Esc key without moving the mouse.

![image](https://github.com/primefaces/primereact/assets/15242152/c3d72e49-97ce-4361-8f2d-c5f621d41624)

![image](https://github.com/primefaces/primereact/assets/15242152/daeb3ab0-ac56-457c-8185-2d91fc238edf)

![image](https://github.com/primefaces/primereact/assets/15242152/2bce0538-ab47-45c7-bfcb-16bf5a36c383)

**Prime React document also says that we support `escape` key.**
![image](https://github.com/primefaces/primereact/assets/15242152/042210af-836e-4841-81a5-aefbc498afdd)

**Also as per WCAG 1.4.13 -**
![image](https://github.com/primefaces/primereact/assets/15242152/2d7533a2-9425-426e-b772-17c2fa6b1519)

### Reproducer

https://stackblitz.com/edit/vitejs-vite-sihhuh?file=src%2FApp.tsx

### PrimeReact version

10.5.3

### React version

17.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Steps to reproduce the behavior

1. Hover over a component that displays a Tooltip
2. Try to dismiss the tooltip by pressing `escape` key

### Expected behavior

`escape` key closes the Tooltip

**Comment by melloware** [comment]
Thanks for reporting.  Definitely a missing feature.

**Comment by akshayaqburst** [comment]
Fixed the issue in my local. Will send the Pull Request soon.
